client/src/components/Dashboard/AddItem.js

```diff
@@ -0,0 +1,135 @@
+import React, { useState, useEffect } from 'react';
+import {
+  makeStyles,
+  MuiThemeProvider,
+  createMuiTheme,
+} from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import Input from '@material-ui/core/Input';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import Typography from '@material-ui/core/Typography';
```

Comment:
Can be rewritten as
``
import {
  Button,
  Input,
  MenuItem,
  Select,
  Typography,
} from '@material-ui/core';
``
This has less code and makes it easier to see that all these functions are from the @material-ui/core module.